site2/docs/reference-configuration.md

```diff
@@ -357,6 +354,22 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 | brokerEntryMetadataInterceptors | Set broker entry metadata interceptors.<br /><br />Multiple interceptors should be separated by commas. <br /><br />Available values:<li>org.apache.pulsar.common.intercept.AppendBrokerTimestampMetadataInterceptor</li><li>org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor</li> <br /><br />Example<br />brokerEntryMetadataInterceptors=org.apache.pulsar.common.intercept.AppendBrokerTimestampMetadataInterceptor, org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor|N/A |
 | enableExposingBrokerEntryMetadataToClient|Whether to expose broker entry metadata to client or not.<br /><br />Available values:<li>true</li><li>false</li><br />Example<br />enableExposingBrokerEntryMetadataToClient=true  | false |
 
+
+### Deprecated Parameters
+The following parameters have been deprecated.
+
+|Name|Description|Default|
+|---|---|---|
+|backlogQuotaDefaultLimitGB|  Use `backlogQuotaDefaultLimitBytes` instead. |-1|
+|brokerServicePurgeInactiveFrequencyInSeconds|  Use `brokerDeleteInactiveTopicsFrequencySeconds`.|60|
+|tlsEnabled|  Use `webServicePortTls` and `brokerServicePortTls` instead. |false|
+|replicationTlsEnabled| Enable TLS when talking with other clusters to replicate messages |false|
+|subscriptionKeySharedEnable|  Whether to enable the Key_Shared subscription.|true|
+|zookeeperServers|  Zookeeper quorum connection string. Use `metadataStoreUrl` instead.  ||
```

Comment:
```suggestion
|zookeeperServers|  Zookeeper quorum connection string. Use `metadataStoreUrl` instead.  |N/A|
```
If no default value, does it make sense to add "N/A" to tell users it does not have a default value rather than empty (tells nothing)?

Reply:
Thanks Yu. Updated.